tests/e2e/component.go

```diff
@@ -61,6 +61,18 @@ func componentTests(componentCmdPrefix string) {
 		})
 	})
 
+	Context("Regression : listing component outside of component directory should fail", func() {
+		It("creates a component from local context, tries to list components from outside and fails", func() {
+			componentName := generateTimeBasedName("nodejs-ex")
+			runCmdShouldPass(componentCmdPrefix + " create nodejs " + componentName + " --git https://github.com/openshift/nodejs-ex")
+			session := runCmdShouldFail("odo component list")
+			Expect(session).To(ContainSubstring("doesn't represent"))
+			Expect(session).To(ContainSubstring("odo component"))
+			// clean up
```

Comment:
would be a good idea to have a single `ContainSubstring` which has the whole message

Reply:
Addressed, pushing shortly